cake.config

```diff
@@ -0,0 +1,14 @@
+; This is the default configuration file for Cake.
```

Comment:
Do we really need this file?

Aren't the values you have specified here the defaults?

Reply:
As discussed on a call, yes, we alter the `SkipVerification` value to true.